mypy/suggestions.py

```diff
@@ -247,7 +253,11 @@ def get_args(self, is_method: bool,
             if default:
                 all_arg_types.append(default)
 
-            if all_arg_types:
+            if len(all_arg_types) == 1 and isinstance(get_proper_type(all_arg_types[0]), NoneType):
```

Comment:
Generally this is good idea, this is consistent to what `stubgen` does. But why do we need to check that length is 1? This is not a set, how do we know all the types are unique there?